actors/builtin/market/market_actor.go

```diff
@@ -132,7 +132,7 @@ func (a Actor) AddBalance(rt Runtime, providerOrClientAddress *addr.Address) *ad
 }
 
 type PublishStorageDealsParams struct {
```

Comment:
Do we want to specify a "max deals per call" limit? It doesn't seem necessary (gas and message sizes should handle this) but we don't have any explicit limits here.

Reply:
Message size limit is probably enough.

 I'm torn about whether as a matter of policy we should set a limit on all such slices anyway, though.